src/Environment/Parameters.yml

```diff
@@ -8,3 +8,4 @@ parameters:
   neighborhoods.kojo.environment.parameters.database_port: 0
   neighborhoods.kojo.environment.parameters.database_name: ''
   neighborhoods.kojo.environment.parameters.lock_prefix: ''
+  neighborhoods.kojo.environment.parameters.kojospace_logging_levels.disabled_job_type: !php/const \Psr\Log\LogLevel::WARNING
```

Comment:
Unfortunately we can't treat this as a YML parameter array (like the log level filter mask) because different actors will be reading from different keys, and that's not possible to do with Symfony's YAML tooling (see the first commit for an idea of what I tried)